### ISSUE TITLE
git_stash segment not working

### ISSUE THREAD

**Issue by shikhanshu** [b-ryan/powerline-shell#384]
I am on release 0.5.0 
```
pip install --user --upgrade powerline-shell
Collecting powerline-shell
  Downloading powerline_shell-0.5.0-py2.py3-none-any.whl
Installing collected packages: powerline-shell
  Found existing installation: powerline-shell 0.4.9
    Uninstalling powerline-shell-0.4.9:
      Successfully uninstalled powerline-shell-0.4.9
Successfully installed powerline-shell-0.5.0
```
Noticed that 'git_stash' segment is new and added it to my config:
```
{
  "segments": [
    "hostname",
    "cwd",
    "git",
    "git_stash",
    "read_only"
  ],
  "hostname": {
    "colorize": true
  },
  "theme": "default"
}
```

Now, whether or not I have anything stashed, it aborts with this error:
```
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/.pkg/software/python/3.6.0/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr2/shikhans/.local/lib/python3.6/site-packages/powerline_shell/segments/git_stash.py", line 22, in run
    self.stash_count = get_stash_count()
  File "/usr2/shikhans/.local/lib/python3.6/site-packages/powerline_shell/segments/git_stash.py", line 17, in get_stash_count
    return pdata[0].count('\n')
TypeError: a bytes-like object is required, not 'str'

Traceback (most recent call last):
  File "/usr2/shikhans/.local/bin/powerline-shell", line 11, in <module>
    sys.exit(main())
  File "/usr2/shikhans/.local/lib/python3.6/site-packages/powerline_shell/__init__.py", line 219, in main
    segment.add_to_powerline()
  File "/usr2/shikhans/.local/lib/python3.6/site-packages/powerline_shell/segments/git_stash.py", line 26, in add_to_powerline
    if not self.stash_count:
AttributeError: 'Segment' object has no attribute 'stash_count'
```

Note, that this crash happens only when changing to a git repo directory. In non-git directories, things are fine. 

Am I missing something? Or is this a bug?

**Comment by shikhanshu** [comment]
Looks like it should be
```
    return pdata[0].count(b'\n')
```
instead of
```
    return pdata[0].count('\n')
```
in git_stash.py?

**Comment by b-ryan** [comment]
Yes your solution worked. This was an issue with Python 3. Just released the fix in 0.5.1
